testlist/x86.dat

```diff
@@ -0,0 +1,6 @@
+# x86_64-elf-gcc from homebrew doesn't seem to
+# provide __udivdi3 etc for -m32
+/x86
```

Comment:
x86 just has two config, it's better to keep it in other.bat to save VM setup time